queries/julia/highlights.scm

```diff
@@ -1,4 +1,8 @@
 (identifier) @variable
+((identifier) @type
+  (match? @type "^[A-Z]"))
+((identifier) @variable
+  (match? @type "^[A-Z_]"))
```

Comment:
This will not  work for most users since `@variable` will not have a highlight. I rather thought about  
```
((identifier) @type
  (match? @type "^[A-Z][a-Z0-9A-Z]"))
````
or 
```
((identifier) @type
  (match? @type "^[A-Z][^_]"))
```
Unfortunately, we don't have unicode regexes.

Reply:
The first one gives an error. The second one works. I'll replace it with the second one.